test/test-spawn.c

```diff
@@ -215,6 +215,54 @@ TEST_IMPL(spawn_fails_check_for_waitpid_cleanup) {
 #endif
 
 
+#ifdef _WIN32
+TEST_IMPL(spawn_requires_elevation) {
+  int r;
+  uv_fs_t req;
+  uv_loop_t* loop = uv_default_loop();
+  char path[1024];
+  size_t path_size = 1024;
```

Comment:
Can you split declaration and assignment? Apparently it produces warnings on older VS